UploadDaemon/Configuration/Config.cs

```diff
@@ -3,17 +3,17 @@
 using System;
 using System.Text.RegularExpressions;
 using System.Linq;
-using static Common.ConfigParser;
+using UploadDaemon.Utils;
 
-namespace Common
+namespace UploadDaemon.Configuration
 {
     /// <summary>
     /// Stores the uploader's config and applies it depending on the
```

Comment:
There's something missing at the end of the comment ^^ We might as well fix it with this pull request ;)